examples/v1beta1/trial-images/enas-cnn-cifar10/requirements.txt

```diff
@@ -1 +1,5 @@
 scipy>=1.7.2
+# TODO (tenzen-y): The line below needs to be deleted after merging https://github.com/kubeflow/katib/pull/1870.
```

Comment:
Ref: https://github.com/kubeflow/katib/pull/1870

Reply:
If we continue to wait for tensorflow-aarch64 v2.9.1, we can not use tensorflow v2.9.1 for Katib v0.14.0. 
So I will update `https://github.com/kubeflow/katib/pull/1870` to install protobuf (>= 3.9.2, < 3.20) to only container image for aarch64 after this PR merging.

Reply:
Should we merge #1870 first ? What do you think?

Reply:
SGTM.
I will send you a review request again after I update https://github.com/kubeflow/katib/pull/1870.

Reply:
Done.